Description: [Django 1.10] Defines TEMPLATES
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-07-25

Index: python-jingo/fake_settings.py
===================================================================
--- python-jingo.orig/fake_settings.py
+++ python-jingo/fake_settings.py
@@ -21,3 +21,24 @@ JINGO_EXCLUDE_APPS = ('django_app',)
 ROOT_URLCONF = 'jingo.tests.urls'
 
 SECRET_KEY = 'jingo'
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [path('jingo/tests/templates'),],
+        'OPTIONS': {
+            'debug': True,
+            'context_processors': [
+                'django.template.context_processors.debug',
+                'django.template.context_processors.request',
+                'django.contrib.auth.context_processors.auth',
+                'django.contrib.messages.context_processors.messages',
+            ],
+            'loaders': [
+                'jingo.Loader',
+                'django.template.loaders.filesystem.Loader',
+                'django.template.loaders.app_directories.Loader',
+            ],
+        },
+    },
+]
