Description: [Django 1.10] django.conf.urls.patterns removed
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-07-25

--- python-jingo-0.9.orig/jingo/tests/urls.py
+++ python-jingo-0.9/jingo/tests/urls.py
@@ -1,9 +1,9 @@
 from __future__ import unicode_literals
 
-from django.conf.urls import patterns
+from django.conf.urls import url
 
 
-urlpatterns = patterns('',
-    (r'^url/(\d+)/(\w+)/$', lambda r: None, {}, "url-args"),
-    (r'^url/(?P<num>\d+)/(?P<word>\w+)/$', lambda r: None, {}, "url-kwargs"),
-)
+urlpatterns = [
+    url(r'^url/(\d+)/(\w+)/$', lambda r: None, {}, "url-args"),
+    url(r'^url/(?P<num>\d+)/(?P<word>\w+)/$', lambda r: None, {}, "url-kwargs"),
+]
