Description: Use six not from django
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-07-25

--- python-jingo-0.9.orig/jingo/ext.py
+++ python-jingo-0.9/jingo/ext.py
@@ -10,7 +10,7 @@ except ImportError:
 from django.core.urlresolvers import reverse
 from django.http import QueryDict
 from django.template.defaulttags import CsrfTokenNode
-from django.utils import six
+import six
 from django.utils.encoding import smart_str
 try:
     from django.utils.encoding import smart_unicode as smart_text
--- python-jingo-0.9.orig/jingo/monkey.py
+++ python-jingo-0.9/jingo/monkey.py
@@ -20,7 +20,7 @@ This patch was originally developed by J
 
 from __future__ import absolute_import, print_function, unicode_literals
 
-from django.utils import six
+import six
 
 
 def __html__(self):
--- python-jingo-0.9.orig/jingo/tests/test_helpers.py
+++ python-jingo-0.9/jingo/tests/test_helpers.py
@@ -7,7 +7,7 @@ import cgi
 from datetime import datetime
 from collections import namedtuple
 
-from django.utils import six
+import six
 from jinja2 import Markup
 try:
     from unittest.mock import patch
--- python-jingo-0.9.orig/jingo/tests/test_monkey.py
+++ python-jingo-0.9/jingo/tests/test_monkey.py
@@ -1,7 +1,7 @@
 from __future__ import unicode_literals
 
 from django import forms
-from django.utils import six
+import six
 
 from jinja2 import escape
 from nose.tools import eq_
